### PR TITLE
Add documentation extension to Jupyter Lab

### PR DESCRIPTION
**Pull request by DidierRLopes**
Add Documentation as an extension. Thanks for the help @piiq :)

Screen Shot 2021-09-11 at 23 51 18

### CODE REVIEW

**Comment by aia** [comment]
I'd name it Gamestonk Docs for clarity.

**Review by aia — APPROVED** [review]

File: jupyterlab/documentation/schema/plugin.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "main": [
+      {
+        "id": "jp-mainmenu-file",
+        "items": [
+          {
+            "type": "submenu",
+            "id": "jp-mainmenu-file-new",
+            "items": [
+              {
+                "command": "jlab-examples:create-new-python-file",
```
Comment (piiq) [inline jupyterlab/documentation/schema/plugin.json:14]: This command should be something like `gamestonk:docs` or smth

File: jupyterlab/documentation/setup.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+# The name of the project
+name = "gamestonk_terminal_documentation"
+
+lab_path = HERE / name.replace("-", "_") / "labextension"
+
+# Representative files that should exist after a successful build
+ensured_targets = [str(lab_path / "package.json"), str(lab_path / "static/style.js")]
+
+labext_name = "@jupyterlab-examples/launcher"
```
Comment (piiq) [inline jupyterlab/documentation/setup.py:20]: This is the extension name. Should change it to smth like `@gamestonk:docs`

File: jupyterlab/documentation/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+const extension: JupyterFrontEndPlugin<void> = {
+  id: 'documentation',
+  autoStart: true,
+  requires: [IFileBrowserFactory],
+  optional: [ILauncher, ICommandPalette],
+  activate: (
+    app: JupyterFrontEnd,
+    browserFactory: IFileBrowserFactory,
```
Comment (piiq) [inline jupyterlab/documentation/src/index.ts:31]: This is not used

File: jupyterlab/documentation/style/Python-logo-notext.svg [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<svg width="22" height="21.884" version="1.0" viewBox="0 0 22 21.884" xmlns="http://www.w3.org/2000/svg">
```
Comment (piiq) [inline jupyterlab/documentation/style/Python-logo-notext.svg:1]: This file is not used

**Review by piiq — COMMENTED** [review]
@DidierRLopes i left some comments in the code during review. Do you see them?

File: jupyterlab/documentation/setup.py (hunk 2 above)
Comment (piiq) [reply jupyterlab/documentation/setup.py:20]: `@gamestonk/docs`

**Comment by DidierRLopes** [comment]
> @DidierRLopes i left some comments in the code during review. Do you see them?

I do see them now! Thank you, will make those changes!

**Pull request merged**